### ISSUE TITLE
Some problems about running environment

### ISSUE THREAD

**Issue by liuyu690806** [danieldimit/methods_6d_pose_estimation#4]
Hello, the author. What is your environment for running SSPE, torch? CUDA cudnn? Python? When I was training, I had the following situation, the program in the loss.backward () quit directly without any error.
[图片](https://user-images.githubusercontent.com/64012147/108455505-566dad80-72a9-11eb-9459-6a2d709aa9d4.png)
[图片](https://user-images.githubusercontent.com/64012147/108455562-70a78b80-72a9-11eb-9b24-bad09f5704b3.png)
